app/src/main/res/drawable/ic_plus_40dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="40dp" android:tint="#D1EEE9"
```

Comment:
Is this icon being used?

Reply:
This was part of my original UI changes, I'll remove it